adafruit_epd/epd.py

```diff
@@ -289,29 +289,29 @@ def line(self, x_0, y_0, x_1, y_1, color):  # pylint: disable=too-many-arguments
         """Draw a line from (x_0, y_0) to (x_1, y_1) in passed color"""
         self._color_dup("line", (x_0, y_0, x_1, y_1), color)
 
-    def text(self, string, x, y, color, *, font_name="font5x8.bin"):
+    def text(self, string, x, y, color, *, font_name="font5x8.bin", size=1):
         """Write text string at location (x, y) in given color, using font file"""
```

Comment:
Please add a sentence about size to the comment. In particular, what are the units?